### PR TITLE
Addresses Chocolatey review comments and fixes logging issue

### LINKED ISSUE #14: Needs license file
The Chocolatey package fails automated validation. It includes binaries, but does not include a license of the software to verify that.

See [LicenseFileMissing](https://github.com/chocolatey/package-validator/wiki/LicenseFileMissing) Chocolatey article.

### LINKED ISSUE #15: Needs verification file
The Chocolatey package fails automated validation. It includes binaries but no way for moderators and the community to verify the files are legitimate.

See [VerificationFileMissing](https://github.com/chocolatey/package-validator/wiki/VerificationFileMissing) Chocolatey article.

### PR DESCRIPTION
**Pull request by gormac**
Minor fix. When running once beacon would still show the verbose logging output `Waiting for 10 seconds.` before exiting.

Addresses Chocolatey moderation review comments. Fixes #14. Fixes #15.

### CODE REVIEW

File: Package/Beacon.nuspec [hunk 1]
```diff
@@ -16,12 +16,17 @@ To get started just run the following command-line:
...
-    </description> 
+or, alternatively using TeamCity guest access and running only once with verbose logging:
+
+`beacon --url=http://yourteamcity.com --guestaccess --runonce --verbose --builds=build_id_1 build_id_2 etc`
+
+    </description>
     <language>en-US</language>
-    <licenseUrl>https://github.com/dennisdoomen/beacon/blob/develop/LICENSE</licenseUrl>    
+    <licenseUrl>https://github.com/dennisdoomen/beacon/blob/master/LICENSE</licenseUrl>
+    <packageSourceUrl>https://chocolatey.org/packages/beacon</packageSourceUrl>
```
Comment (dennisdoomen) [inline Package/Beacon.nuspec]: This should point to `https://github.com/dennisdoomen/beacon`.

Comment (dennisdoomen) [reply Package/Beacon.nuspec]: Probably also good to add  `bugTrackerUrl` and have it point at `https://github.com/dennisdoomen/beacon/issues`

**Comment by gormac** [comment]
Hold on. Package versioning is wrong in the build.

**Comment by gormac** [comment]
Should be good now.

**Comment by gormac** [comment]
@dennisdoomen, this PR could use some love and attention in order to get Beacon 1.2.0 released on Chocolatey.

**Pull request merged**